cmd/juju/application/bundle.go

```diff
@@ -1390,6 +1390,18 @@ func buildModelRepresentation(
 			return nil, errors.Errorf("unexpected tag kind for annotations: %q", kind)
 		}
 	}
+	// Add in the model sequences.
+	sequences, err := apiRoot.Sequences()
+	if err == nil {
```

Comment:
Agree with falling back to not using sequences, but if err is anything other than not implemented, or even in all not nil cases, I think it warrants a warning so we are explicitly not surprising the user. Once we release a version of bundle deploy with the new behaviour, it would be bad to just transparently deploy to "wrong" machine ids.